index.bs

```diff
@@ -1846,47 +1843,33 @@ on to the next ad in the current ad pod).
 
 The player must follow this workflow for loading an ad.
 
-1. The player must create an iframe element for the SIMID creative. The player
-	iframe should start out hidden. The iframe should be capable of executing
-	javascript and loading resources.
-2. The player starts listening on the window that is the parent of the iframe
-	for messages from the creative.
-3. The player sets the src element of the iframe to the url provided by
-	the creative's VAST InteractiveCreativeFile element. The player should
-	assume this will be a cross domain iframe.
-4. The player waits until the creative inializes a session.
-	[[#establish-session]] The player responds with a resolve message.
-	The resolve message includes the correlator that must be present in
-	all messages going forward.
-5. The player then sends a [[#simid-player-init]] message with all relevant parameters.
-	The player waits until the creative responds with resolve.  If the
-	creative responds with reject, the player should immediately unload
-	the creative's iframe.
-6. Where possible the player should wait until both the creative has responded
-	to the [[#simid-player-init]] and the video is ready to start playing. Ready
-	to start playing means the first frame will show and playback will continue.
-7. When the video is started the player sends a [[#simid-player-startCreative]] message.
-	The player must overlay the creative iframe over the video element exactly.
-	The player must make the creative visible. The creative should respond to
-	this message immediately with [[#simid-player-startCreative-resolve]].
+1. The player creates an iframe for the SIMID interactive component. The iframe should start hidden. While hidden, the iframe must be capable of executing javascript and loading resources.
+1. The player starts listening to the message event on the window that is the parent of the creative iframe.
+1. The player sets the iframe src attribute to the URL provided by the creative's VAST `<InteractiveCreativeFile/>` element. The player must assume the iframe is cross-origin.
+1. The player waits until the creative initializes a session and posts `createSession` message (see [[#protocol-session-layer]]). The player responds to the session initialization with a `resolve` message. 
+
+1. The player sends a [[#simid-player-init]] message with relevant parameters. The player waits until the creative responds with [[#simid-player-init-resolve]]. If the creative responds with [[#simid-player-init-reject]], the player should unload the creative's iframe.
+
+1. Where possible, to synchronize media playback and the creative UI, the player should wait until both the creative has responded to the [[#simid-player-init]] with [[#simid-player-init-resolve]] and the media is ready to play. Media readiness means sufficient payload arrived, and the first frame shows.
```

Comment:
I would just remove the last sentence - I'm not sure it adds something that someone can implement.